security-oauth2/build.gradle.kts

```diff
@@ -1,13 +1,14 @@
 plugins {
-    id "io.micronaut.build.internal.security-module"
+    id("io.micronaut.build.internal.security-module")
 }
 
 dependencies {
     annotationProcessor(mn.micronaut.graal)
     annotationProcessor(mnSerde.micronaut.serde.processor)
     annotationProcessor(projects.micronautSecurityAnnotations)
     annotationProcessor(mnValidation.micronaut.validation.processor)
-    api(mnValidation.micronaut.validation)
+    api(mnValidation.validation) //  // jakarta.validation:jakarta.validation-api
```

Comment:
What's the comment for here?

Reply:
it is totally alien to me that `mnValidation.validation` means `jakarta.validation:jakarta.validation-api`.  If instead of`mnValidation.validation` it was `mnValidation.jakarta.validation.api`I would have not added the comment.